client/src/App.js

```diff
@@ -18,7 +18,7 @@ const App = () => {
 	const [energisers, setEnergisers] = useState([]);
 
 	useEffect(() => {
-		const socket = io("https://cyf-booster.herokuapp.com");
+		const socket = io("https://cyf-booster.herokuapp.com/game");
```

Comment:
You added it again 😆 
It won't work in Heroku with this link. You have to remove it I suggest.

Reply:
removed.